roles/softserve/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+user: softserve
```

Comment:
Could we have _softserve ? So this avoid risk of  clash in the future with a non system account ?